wikitextparser/_config.py

```diff
@@ -190,6 +204,8 @@ def regex_pattern(words: _Iterable[str]) -> bytes:
     'REVISIONYEAR',
     'ROOTPAGENAME',
     'ROOTPAGENAMEE',
+    'SERVER',
+    'SERVERNAME',
```

Comment:
There are more of them, e.g. ``{{SITENAME}} ``, but I can add them later.

Reply:
Added all the missing technical metadata!